src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/PropertyNamesShouldNotMatchGetMethods.cs

```diff
@@ -50,8 +50,9 @@ private static void AnalyzeSymbol(SymbolAnalysisContext context)
             string identifier;
             var symbol = context.Symbol;
 
-            // Bail out if the method/property is not exposed (public, protected, or protected internal)
-            if (!symbol.IsExternallyVisible())
+            // Bail out if the method/property is not exposed (public, protected, or protected internal) by default
+            var configuredVisibilities = context.Options.GetSymbolVisibilityGroupOption(Rule, SymbolVisibilityGroup.Public, context.CancellationToken);
```

Comment:
why not using matchesConfiguredVisiblity ? if default being different is issue, can the method changes to accept custom default?

Reply:
Just optimization as we need to use `configuredVisibilities` again below.